cmd/earthly/project_cmds.go

```diff
@@ -0,0 +1,332 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/earthly/earthly/cloud"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli/v2"
+)
+
+func (app *earthlyApp) projectCmds() []*cli.Command {
+	return []*cli.Command{
+		{
+			Name:        "ls",
+			Usage:       "List all projects that belong to the specified organization",
+			Description: "List all projects that belong to the specified organization",
+			UsageText:   "earthly project [--org <organization-name>] ls",
+			Action:      app.actionProjectList,
+		},
+		{
+			Name:        "rm",
+			Usage:       "Remove an existing project from the organization",
+			Description: "Remove an existing project from the organization",
+			UsageText:   "earthly project [--org <organization-name>] rm <project-name>",
+			Action:      app.actionProjectRemove,
+		},
+		{
+			Name:        "create",
+			Usage:       "Create a new project in the specified organization",
+			Description: "Create a new project in the specified organization",
+			UsageText:   "earthly project [--org <organization-name>] create <project-name>",
+			Action:      app.actionProjectCreate,
+		},
+		{
+			Name:        "member",
+			Usage:       "Create, list, and edit project members",
+			Description: "Create, list, and edit project members",
+			UsageText:   "earthly project member (ls|rm|add|update)",
+			Subcommands: []*cli.Command{
+				{
+					Name:        "add",
+					Usage:       "Add a new member to the specified project",
+					Description: "Add a new member to the specified project",
+					UsageText:   "earthly project [--org <organization-name>] member add <project-name> <user-id-or-email> <permission>",
+					Action:      app.actionProjectMemberAdd,
+				},
+				{
+					Name:        "rm",
+					Usage:       "Remove a member from the specified project",
+					Description: "Remove a member from the specified project",
+					UsageText:   "earthly project [--org <organization-name>] member rm <project-name> <user-id>",
+					Action:      app.actionProjectMemberRemove,
+				},
+				{
+					Name:        "ls",
+					Usage:       "List all members in the specified project",
+					Description: "List all members in the specified project",
+					UsageText:   "earthly project [--org <organization-name>] member ls <project-name>",
+					Action:      app.actionProjectMemberList,
+				},
+				{
+					Name:        "update",
+					Usage:       "Update the project member's permission",
+					Description: "Update the project member's permission",
+					UsageText:   "earthly project [--org <organization-name>] member update <project-name> <user-id> <permission>",
+					Action:      app.actionProjectMemberUpdate,
+				},
+			},
+		},
+	}
+}
+
+func (app *earthlyApp) actionProjectList(cliCtx *cli.Context) error {
+	app.commandName = "projectList"
```

Comment:
Note that I noticed that these were missing for project commands so I added them in, @mikejholly. These are needed for identifying which command was issued for analytics.